source/ghost-text.js

```diff
@@ -172,6 +172,7 @@ class GhostTextField {
 					}
 				}));
 				this.field.dispatchEvent(new KeyboardEvent('keyup'), eventOptions);
+				this.field.dispatchEvent(new Event('change', eventOptions));
```

Comment:
It doesn't look like `change` is fired for every keystroke. You can use `monitorEvents(document.querySelector('textarea'))` to see this in the console:

<img width="777" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/129487120-c26c92b2-2321-40fc-aadf-9dbd486c60ee.png">